docs/content/plugins/hashicorp.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Hashicorp plugin
+description: Integrate Porter with Hashicorp
+---
+
+<img src="/images/mixins/hashicorp.png" class="mixin-logo" style="width: 300px"/>
```

Comment:
The image should go in `/images/plugins` and this link updated with the new location.